src/core.jl

```diff
@@ -253,13 +231,14 @@ function mapToGigaSOM(som::Som, data;
                       knnTreeFun = BruteTree,
                       metric = Euclidean())
 
-    data = convertTrainingData(data)
+    data = Matrix{Float64}(data)
 
     if size(data,2) != size(som.codes,2)
         @error "Data dimension ($(size(data,2))) does not match codebook dimension ($(size(som.codes,2)))."
+        error("Data dimensions do not match")
     end
 
-    dInfo= distribute_darray(:mappingDataVar, distribute(data))
+    dInfo= distribute_array(:GigaSOMmappingDataVar, data, workers())
```

Comment:
what is GigaSOMmappingDataVar? Is it a global var?

Reply:
Yes, global in the distributed sense. The name is this long and prefixed in order to avoid collision as much as possible.